src/Antiforgery/src/RequireAntiforgeryTokenAttribute.cs

```diff
@@ -7,8 +7,9 @@ namespace Microsoft.AspNetCore.Antiforgery;
 /// An attribute that can be used to indicate whether the antiforgery token must be validated.
 /// </summary>
 /// <param name="required">A value indicating whether the antiforgery token should be validated.</param>
+/// <param name="requestTokenSource">A value indicating locations in request to look for antiforgery token.</param>
 [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method)]
-public class RequireAntiforgeryTokenAttribute(bool required = true) : Attribute, IAntiforgeryMetadata
+public class RequireAntiforgeryTokenAttribute(bool required = true, AntiforgeryRequestTokenSource? requestTokenSource = null) : Attribute, IAntiforgeryMetadata
 {
     /// <summary>
     /// Gets or sets a value indicating whether the antiforgery token should be validated.
```

Comment:
BTW, it looks like documentation bug to me as there is no property setter to make it "Gets or sets"